oneview_redfish_toolkit/api/service_root.py

```diff
@@ -0,0 +1,60 @@
+#!./redfish-venv/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import collections
+from oneview_redfish_toolkit.api.redfish_json_validator import \
+    RedfishJsonValidator
+
+
+class ServiceRoot(RedfishJsonValidator):
+    '''Creates a hardcoded ServiceRoot redfish dict
+
+        Populates self.redfish with a hardcoded ServiceRoot values for
+        a BladeSystem
+
+    '''
+
+    def __init__(self, schema_obj):
+        '''Constructor
+
+            Populates self.redfish with a hardcoded ServiceRoot values for
+            a BladeSystem
+        '''
+        super().__init__(schema_obj)
+        self.redfish["@odata.type"] = "#ServiceRoot.v1_2_0.ServiceRoot"
+        self.redfish["Id"] = "RootService"
+        self.redfish["Name"] = "Root Service"
+        self.redfish["RedfishVersion"] = "1.2.0"
+        self.redfish["UUID"] = "85775665-c110-4b85-8989-e6162170b3ec"
```

Comment:
I might be getting ahead of myself here but I thought UUID was supposed to be populated from this OV REST call: https://{appl}/rest/appliance/nodeinfo/version